gcloud/language/client.py

```diff
@@ -54,7 +54,7 @@ def document_from_text(self, content, **kwargs):
 
         :rtype: :class:`Document`
         :returns: A plain-text document bound to this client.
-        :raises: :class:`TypeError` if ``doc_type`` is passed as a
+        :raises: :class:`~exceptions.TypeError` if ``doc_type`` is passed as a
```

Comment:
Yes but sadly in https://docs.python.org/2/objects.inv it shows up as `exceptions.TypeError`

PS I hope you weren't assuming I didn't know `TypeError` was a builtin